inst/markdown/compareScenarios2/cs2_main.Rmd

```diff
@@ -238,6 +238,9 @@ data <-
         unit = "billion US$2005/yr",
         value = value / 1000)
   )
+# backwards compatibility for https://github.com/pik-piam/remind2/pull/579
+data <- data %>%
```

Comment:
Why are you changing back the renaming here?

Reply:
Ah, now I understand. Once again, I question this approach to backwards compatibility. If you have old runs, why not use the fitting remind2? These kind of backwards compatibility adjustments come with technical debt and clutter the code. So if you are certain, it has a real life use, keep it. Otherwise, drop it.

Reply:
I'm not changing back. If I get old data that still has `|Elec|` variables (for example all the AMT comparisons with older runs, or comparing an old release version to today's runs), I convert them into `|Electricity|` such that they can be plotted.